### PR TITLE
asset query

### PR DESCRIPTION
**Pull request by ialberquilla**
## Description

Adds new command to query assets using elasticsearch query

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [x] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

File: test/integration-tests/Assets.test.ts [hunk 1]
```diff
@@ -110,6 +110,22 @@ describe('Assets e2e Testing', () => {
...
+
+    const queryAssetCommand = `${baseCommands.assets.queryAsset} ${query} `
+    console.debug(`COMMAND: ${queryAssetCommand}`)
+
+    const searchStdout = execCommand(queryAssetCommand, execOpts)
+    console.log(`STDOUT: ${searchStdout}`)
+
+    const numberResults = parseNumberResultsFromSearch(searchStdout)
+    console.log(`Number of Results: ${numberResults}`)
+    expect(numberResults === null ? false : Number(numberResults) > 0)
```
Comment (r-marques) [inline test/integration-tests/Assets.test.ts]: this only tests that the command is executed but not if it's working correctly

Comment (ialberquilla) [reply test/integration-tests/Assets.test.ts]: fixed now

Comment (r-marques) [reply test/integration-tests/Assets.test.ts]: I mean we should do a real query and expect that a result is returned

Comment (ialberquilla) [reply test/integration-tests/Assets.test.ts]: ok, searching by attribute name now and checking that results > 0

**Review by r-marques — APPROVED** [review]

**Pull request merged**